performance/README.md

```diff
@@ -0,0 +1,135 @@
+# neo4j-streams performance test
+
+This part of the project aims to give a standard way to understand the performances of the platform and highlight the configuration results.
+
+
+
+Currently, it's available only the test of neo4j to neo4j configuration, in remote too.
+
+## Setup
+
+The basic unit of work used to test the performances needs a specific cypher part in order to get the timestamp of creation and receiving of the event. The elapsed time is calculated using these timestamps through cypher queries.
+
+### Producer
+
+- neo4j 3.4.7 community edition
+- APOC 3.4.0.3
+- 8 GB heap memory
+- CREATE INDEX ON :Performance(group)
+- default neo4j-streams configuration (without filtering)
+- kafka.batch.size=16384
+
+The  creation query used by the tester:
+
+	UNWIND range(1,{msg}) as ran
+	CREATE (n:Performance)
+	SET n.group = {uuid}, n.creation_time = apoc.date.currentTimestamp()
+	RETURN min(n.creation_time) as creation_time
+### Consumer
+
+- neo4j 3.4.7 community edition
+- APOC 3.4.0.3
+- 2 GB heap memory
+- CREATE INDEX ON :Performance(group)
+- default neo4j-streams configuration
+- kafka.max.poll.records=16384
+
+The acquisition query:
+
+```properties
+streams.sink.topic.neo4j=WITH event.value.payload AS payload, event.value.meta AS meta CALL apoc.do.case( [\
```

Comment:
this can be a regular "WHERE" filter :)

Reply:
in real scenario, apoc.do.case may be more useful to split the cases